panther-labs/panther
# rule error function prefix

- Add version to all tracking events (#1720)
- Tests for tracking utils (#1695)
- Redesign Rules page 1/3 (#1710)
- Add panther version check to opstools gluesync and gluerecover (#1723)
- Make StackOutputs() return error (#1721)
- Added warning (#1730)
- Include title() & dedup() output in API response when testing a Rule (#1625)
- Use camelCase for rule engine JSON responses
- Prefix rule result error message with the function that raised the error
